ocsmesh/driver.py

```diff
@@ -71,14 +72,24 @@ def __init__(
         """
         geom can be SizeFunction or PlanarStraightLineGraph instance.
         """
+        warnings.warn(
+            "JigsawDriver is deprecated and will be removed in a future release. "
+            "Please use 'ocsmesh.driver.MeshDriver' with engine_name='jigsaw'." 
+            "We strongly suggest the adoption of gmsh and the driver engine",
```

Comment:
Maybe rephrase to "we suggest considering Gmsh for your workflow as the that's going to be the default for OCSMesh in the upcoming versions." or something like that